bap_ida_python/loader/installer.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python
+"""Loader for setting up IDA to successfully import all BAP plugins."""
+
+
+def _multi_line_string_clean(s):
+    s_split = s.split('\n')[1:]
+    for i, c in enumerate(s_split[0]):
+        if not c.isspace():
+            left_space_count = i
+            break
+    return '\n'.join(x[left_space_count:] for x in s_split)
+
+
+def get_bap_path():
+    """Return path of bap executable or None."""
+    from subprocess import check_output, CalledProcessError
+    import os
+    try:
+        bap_path = check_output(['which', 'bap']).strip()
+    except OSError:  # Cannot run 'which' command
+        print "[!] Could not determine bap path automatically."
+        print "[ ] Please enter path to bap executable (including /bap): "
+        bap_path = input('  BAP Path: ').strip()
+    except CalledProcessError:  # 'which' could not find 'bap'
+        print "[!] Unable to locate bap. Please install it and ensure that it"
+        print "        is on your path, and `bap --version` works."
+        return None
+    if os.path.isfile(bap_path) and bap_path.endswith('/bap'):
+        return bap_path
+    else:
+        print "[!] Unable to locate bap at {}".format(bap_path)
+        return None
+
+
+def install_bap_config():
```

Comment:
can we do this without a config file at this stage?

Configuration is a complex feature, that should be designed carefully, not just added on per need basis. We should think about the general approach for storing static configuration data, that will work uniformly across all plugins and parts of the platform. 

Also, it looks really weird, that some non-essential component, is responsible for installation of BAP config. The config should be installed by the BAP, and owned by it. 

Finally, I feel really bad about establishing a communication channel between two software components via a configuration file. A program shouldn't modify the config file, it is the realm of a human. It can be generated by a program, but then left to a human. Even worser, if one program is changing a configuration file of another program.